src/coreclr/ToolBox/superpmi/superpmi-shared/icorjitinfoimpl.h

```diff
@@ -101,8 +101,9 @@ CorInfoIntrinsics getIntrinsicID(
 bool isIntrinsicType(
           CORINFO_CLASS_HANDLE classHnd);
 
-CorInfoUnmanagedCallConv getUnmanagedCallConv(
-          CORINFO_METHOD_HANDLE method);
+CorInfoCallConvExtension getEntryPointCallConv(
```

Comment:
Nit: I would keep the original name. Or at least have "unmanaged" mentioned in the name.

Reply:
I'll change it back to the original name and add a comment about how it can return `CorInfoCallConvExtension::Managed`	 when the call doesn't use an unmanged calling convention.